src/index.html

```diff
@@ -21,7 +21,449 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="is-active nav__link"
```

Comment:
The class 'is-active' is not defined anywhere in your CSS. If it is not necessary, consider removing it to avoid confusion.